lib/process-services-cloud/src/lib/process/services/process-cloud.service.ts

```diff
@@ -63,11 +62,8 @@ export class ProcessCloudService extends BaseCloudService implements ProcessClou
         if (appName || appName === '') {
             const url = `${this.getBasePath(appName)}/rb/v1/process-definitions`;
 
-            return this.get(url).pipe(
-                map((res: any) => res.list.entries.map((processDefs) => new ProcessDefinitionCloud(processDefs.entry)))
-            );
+            return this.get(url).pipe(map((res: any) => res.list.entries.map((processDefs) => new ProcessDefinitionCloud(processDefs.entry))));
```

Comment:
Same for `any`

Reply:
this needs investigations on what exactly is the return type, as it invokes the plain http there, not the js-api.